ideascube/conf/base.py

```diff
@@ -283,3 +283,6 @@
 
 SESSION_COOKIE_AGE = 60 * 60  # Members must be logged out after one hour
 BACKUP_FORMAT = 'zip'  # One of 'zip', 'tar', 'bztar', 'gztar'
+
+CATALOG_CACHE_BASE_DIR = '/tmp/ideascube-catalog'
```

Comment:
If `/tmp` is mounted as `tmpfs`, and if deployments put their cache in there... then yes.

It's probably not the right location for the cache, for sure.

How about `/var/cache/ideascube` ? (but then, we get into permission-trouble territory :stuck_out_tongue_winking_eye: )